packages/calendar/src/components/week-grid/time-grid-event.tsx

```diff
@@ -59,6 +59,11 @@ export default function TimeGridEvent({
     const localizedStartTime = toJSDate(start).toLocaleTimeString(
       ...localizeArgs
     )
+
+    if (start === end) {
```

Comment:
I think we should also have a unit test for this.

Reply:
Done, let me know if it's not enough